roles/lib_utils/src/class/repoquery.py

```diff
@@ -125,6 +145,9 @@ def repoquery(self):
         else:
             rval['package_found'] = False
 
+        if self.ignore_excluders:
```

Comment:
The `tmp_file` should be deleted once the `with` clause is exited. Which is before `repoquery_cmd = self.build_cmd()` command. Or am I missing something here?